app/src/main/res/layout/partial_feedback_result.xml

```diff
@@ -84,44 +84,89 @@
             tools:visibility="visible">
 
             <View
-                android:id="@+id/line"
+                android:id="@+id/otherPredictionsSeparator"
```

Comment:
In this case, it should be otherPredictionsSeparator, because this is the line above "Other predictions" of our local model.